apps/gauzy/src/app/pages/settings/oauth-clients/oauth-client-secret-dialog/oauth-client-secret-dialog.component.html

```
@@ -0,0 +1,42 @@
+	</nb-card-header>
+
+	<nb-card-body>
+		<nb-alert status="warning" class="warning">
+			{{ 'OAUTH_CLIENTS.SECRET_DIALOG.WARNING' | translate }}
+		</nb-alert>
+
+		<div class="credential">
+			<label>{{ 'OAUTH_CLIENTS.SECRET_DIALOG.CLIENT_ID' | translate }}</label>
+			<div class="credential-row">
+				<code>{{ client.clientId }}</code>
+				<button nbButton size="small" status="primary" ghost (click)="copyClientId()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)